infra/logging/src/main/java/com/simprints/infra/logging/Simber.kt

```diff
@@ -1,8 +1,8 @@
 package com.simprints.infra.logging
 
+import co.touchlab.kermit.Logger
```

Comment:
I kind of assumed that we would add something to Simber, not completely change the underlying library. Couldn't we just add a new Tree to Timber?

Reply:
We would need to rewrite the whole file writing and rotating logic ourselves, which could cause more issues and work.

Reply:
Yeah, I expected that we would plug-in Kermit as one more `CrashReportingTree`. Appreciate that makes things much more complex.
How confident are you switching to Kermit won't cause any major issues in prod? Timber, for all it's flaws, has served us well for a long time.

Reply:
It should be fine since Kermit is developed by one of the largest mobile dev agencies online and they would be directly interested in supporting it.

Reply:
Checking it's GitHub - looks well supported, indeed. Frequent releases and just 3 open issues.